compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.cpp

```diff
@@ -1029,6 +1032,78 @@ bool TargetAttr::supportsSyncMMAOps() const {
   return false;
 }
 
+//===----------------------------------------------------------------------===//
+// Lowering Config Attributes
+//===----------------------------------------------------------------------===//
+
+static constexpr char kWorkgroupLevelName[] = "workgroup";
```

Comment:
nit: in the global scope, contexpr implies const which implies static. Also we can make it `StringLiteral`.
```suggestion
constexpr StringLiteral kWorkgroupLevelName = "workgroup";
```

same below